### ISSUE TITLE
Location of gatling.conf

### ISSUE THREAD

**Issue by r-alukhanov** [gatling/gatling#2915]
According to documentation the configuration file "gatling.conf" is located in the "conf" folder.

http://gatling.io/docs/2.0.0-RC2/general/configuration.html

It is not specified, where the "conf" folder is located. Relative to the current directory? Or may be it must be a subfolder/package of the classpath?

As far I could unterstand the implementation, the "gatling.conf" is always read from the root of the class loader (for both "gatling-defaults.conf" and "gatling.conf" configuration file).

The object `io.gatling.core.config.GatlingConfiguration` (which seems to be responsible for the loading of the configuration) loads the file "gatling.conf" from the classpath without possibility to relocate it.

```
val classLoader = getClass.getClassLoader

val defaultsConfig = ConfigFactory.parseResources(classLoader, GatlingDefaultsConfigFile)
val customConfig = ConfigFactory.parseResources(classLoader, GatlingConfigFile)
val propertiesConfig = ConfigFactory.parseMap(props)

val config = configChain(ConfigFactory.systemProperties, customConfig, propertiesConfig, defaultsConfig)
```

The configuration parameter "configFolder" of the gatling maven plugin seems to have no effect.

Either the documentation or the code of `GatlingConfiguration` object seems to be wrong.

**Comment by slandelle** [comment]
> As far I could unterstand the implementation, the "gatling.conf" is always read from the root of the class loader (for both "gatling-defaults.conf" and "gatling.conf" configuration file).

Yes

> The configuration parameter "configFolder" of the gatling maven plugin seems to have no effect.

It adds this `configFolder` to the classpath, see [AbstractGatlingMojo](https://github.com/gatling/gatling-maven/blob/master/gatling-maven-plugin/src/main/java/io/gatling/mojo/AbstractGatlingMojo.java#L83-L87). If it doesn't work the way you expect, it's probably because you have multiple `gatling.conf` files in your classpath.

**Comment by r-alukhanov** [comment]
What is with the "conf" folder? Bug in the documentation?

**Comment by slandelle** [comment]
You have a conf folder when you use the zip bundle.

**Comment by SmotrovaLilit** [comment]
> You have a conf folder when you use the zip bundle.


It would be great to add this information to the documentation.

**Comment by slandelle** [comment]
@SmotrovaLilit 

We duplicate this file everywhere:

* in the bundle (packaged from https://github.com/gatling/gatling/blob/master/gatling-core/src/main/resources/gatling-defaults.conf)
* in the maven demo: https://github.com/gatling/gatling-maven-plugin-demo/blob/master/src/test/resources/gatling.conf
* in the gradle demo: https://github.com/gatling/gatling-gradle-plugin-demo/blob/master/src/gatling/resources/gatling.conf
* in the sbt demo: https://github.com/gatling/gatling-sbt-plugin-demo/blob/master/src/test/resources/gatling.conf
* in the maven archetype: https://github.com/gatling/gatling-highcharts-maven-archetype/blob/master/src/main/resources/gatling.conf
* in the giter8 template: https://github.com/gatling/gatling.g8/blob/master/src/main/g8/src/test/resources/gatling.conf

Could you please explain how you ended up missing it?

**Comment by SmotrovaLilit** [comment]
@slandelle 
I put the file in the resource directory and it didn't work. I thought it was because I put the file in the wrong place and wanted the documentation there https://gatling.io/docs/current/general/configuration/ to clearly say where to put the file. 

Then I found out that the problem was that I used the old version, and there were other parameters. This is my fault, sorry for my suggestion
